### PR TITLE
Branch mengzhe check duplicate tele handle email

### LINKED ISSUE #171: [PE-D] Duplicate telegram handles accepted when adding student
Different person with same telegram handles are allowed to be added even though telegram handles are unique
Error message should be thrown instead

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: auhc99/ped#4

### PR DESCRIPTION
**Pull request by charliemoweng**
Fix #171

### CODE REVIEW

**Review by tanjoewel — APPROVED** [review]
LGTM!

**Comment by Ruofan1023** [comment]
LGTM

File: src/main/java/seedu/address/logic/commands/AddStudentCommand.java [hunk 1]
```diff
@@ -92,6 +107,40 @@ public CommandResult execute(Model model) throws CommandException {
...
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Checks if an Email is a duplicate of an existing one in TAB.
+     *
+     * @param email The Email to be verified.
```
Comment (felissaf) [inline src/main/java/seedu/address/logic/commands/AddStudentCommand.java:130]: param module is missing?

**Review by felissaf — COMMENTED** [review]
Looks goood!

File: src/main/java/seedu/address/logic/commands/AddStudentCommand.java (hunk 1 above)
Comment (charliemoweng) [reply src/main/java/seedu/address/logic/commands/AddStudentCommand.java:131]: Ok updated. Thanks for pointing it out!

**Pull request merged**